source/Ship.cpp

```diff
@@ -2336,6 +2444,10 @@ void Ship::DoGeneration()
 				heat -= activeCooling;
 		}
 	}
+	else if(!this->HasSpriteFor(BodyState::DISABLED))
+	{
+		this->PauseAnimation();
+	}
```

Comment:
```suggestion
		this->PauseAnimation();
```
Single line code doesn't need braces